pom.xml

```diff
@@ -1497,6 +1497,40 @@
                 <module>assembly</module>
             </modules>
         </profile>
+        <profile>
+            <id>dbaas</id>
+            <activation>
+                <property>
+                    <name>env</name>
+                    <value>dbaas</value>
+                </property>
+            </activation>
+            <properties>
+                <cdh.parcel.version>5.14.0</cdh.parcel.version>
```

Comment:
Just noticed all the uses of `5.14.0` and wonder if it could be somehow reused through the profile.